.github/workflows/mongodb-example.yml

```diff
@@ -1,4 +1,4 @@
-name: Mongodb example pipeline
+name: MongoDB example pipeline
```

Comment:
This is a good point, as the generator does not distinguish this notation including initials. I'll take a note to include a way to provide a variant that overrides the titled names